packages/home-connector/src/transport/worker-connector.ts

```
@@ -237,96 +283,199 @@ export function createWorkerConnector(input: {
+							extra: createSocketEventContext({
+								config: input.config,
+								connectionAttempt,
+								consecutiveReconnects,
 							}),
-						)
+						})
+						console.error(`Home connector error: ${value.message}`)
+						return
+					case 'server.ack':
+						hasReportedSocketIssue = false
+						const previousConsecutiveReconnects = consecutiveReconnects
```

Biome (2.4.12)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)